### DISCUSSION TITLE
ME Swapper/Swapper Extended

### DISCUSSION THREAD

**Discussion by Blue-R6** [runelite/runelite#16139] [Q&A] [answered]
Hi,

I think it would be cool to have an option in Menu Entry Swapper, or MES Extended (probably the more fitting plugin) to claim slime from Robin.

Sincerely, HCIM's

**Comment by Nightfirecat** [comment, marked as answer]
The vanilla menu entry swapper plugin is already capable of this via customizable NPC swaps. Shift+right-click Robin and swap either his left-click or shift-click option to "Claim-slime"

**Reply by Blue-R6** [reply]
That was fast, and you are correct! Thanks
